### ISSUE TITLE
Syncing my sequencer/ synth developments with your repo

### ISSUE THREAD

**Issue by goofy2k** [ncassetta/NiCMidi#14]
I see some tags about reverting something done by me.  This probably has to do with a try by me on staying synced with changes in your repo. Actually I wasn't aware that I changed / requested something on your side.

As I don't quite understand how to stay synced, I decided to postpone that.  It is better to stay focused on the "real work".

And an update on what I am currently doing:

I am currently adding more and more implementations of midi commands to my sequencer app and my synth app. My synth app now responds to a number of Control Changes (Midi Bn) and Program Changes (Midi Cn).  With the program changes, I can activate groups of sound parameters (in fact "instrument presets").  I also added a rough way to edit individual parameters in a basic GUI. To make editing more convenient I am now implementing a bluetooth / NimBLE output to the synth to communicate current parameter settings to my GUI . This also involves creation of a NimBLE input port in the sequencer app. So I have to extract initialisation of the NimBLE server from the part that I added to NiCMidi.  In other words:  I move creation of the NimBLE serverfrom NiCMidi  to a "system level" to have it available even before Manager has to open the Midi output port.

**Comment by ncassetta** [comment]
Yes, you sent me a pull request, i.e. a request to merge your changes in my repo. As I also don't have a deep knowledge of github, I pulled it into the master, then I realized that it would be better to put your changes in a new branch. So I moved them to the branch FCKX and reverted the changes in the master. So the master is to the ver 1.1.0

**Comment by goofy2k** [comment]
OK. I will probably don't touch it. You could also delete it / clean up.

After  I consolidate my current work, I will first practice a bit with
Github and then try to set up some way to follow your progress and
hopefully cooperate further.



Op vr 11 feb. 2022 20:53 schreef Nicola Cassetta ***@***.***>:
